myapp/app/controllers/tasks_controller.rb

```diff
@@ -66,13 +89,23 @@ def destroy
     redirect_to root_path
   end
 
+  def get_status_map
+    {
+      Constants::STATUS_NOT_STARTED => I18n.t('status_not_started'),
```

Comment:
Is this actually NOT mysql data type enum, and Rails own type to wrap the integer, correct? I'm not using mysql enum on purpose to avoid cumber some migration problems (and just my personal preference...)
Let me do some research about the Rails enum, and will update the code.

Reply:
it's `ActiveRecord::Enum`. please  check  this document.
https://api.rubyonrails.org/classes/ActiveRecord/Enum.html

Reply:
Thanks, so I updated to use the Enum. Please check latest commit.